addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -497,6 +516,7 @@ function factory(dependencies) {
         thread: many2one('mail.thread', {
             compute: '_computeThread',
             dependencies: [
+                'activeMobileNavbarTabId',
                 'isThreadPinned',
                 'thread',
             ],
```

Comment:
`_computeThread` uses `env.messaging`, `env.messaging.inbox`, `env.messaging.device.isMobile` and `this.thread.model`, which all of them are missing in dependencies:

```js
dependencies: [
   'activeMobileNavbarTabId',
   'deviceIsMobile',
   'isThreadPinned',
   'messaging',
   'messagingInbox',
   'thread',
   'threadModel',
],
```

Will need to define related fields:
```js
device: many2one('mail.device', {
   related: 'messaging.device',
}),
deviceIsMobile: attr({
   related: 'device.isMobile'
}),
messagingInbox: many2one('mail.thread', {
   related: 'messaging.inbox',
}),
threadModel: attr({
   related: 'thread.model',
}),
```

_Yeah... this is something we want to improve ASAP... not easy feat..._